### PR TITLE
Add mainnet contract addresses for registrar and price oracle

### PR DESCRIPTION
**Pull request by dneilroth**
**What changed? Why?**
Add mainnet contract addresses for registrar and price oracle

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/web/src/addresses/usernames.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const USERNAME_L2_RESOLVER_ADDRESSES: AddressMap = {
 
 export const USERNAME_REGISTRAR_CONTROLLER_ADDRESSES: AddressMap = {
   [baseSepolia.id]: '0x49aE3cC2e3AA768B1e5654f5D3C6002144A59581',
-  [base.id]: '0x',
+  [base.id]: '0x4cCb0BB02FCABA27e82a56646E81d8c5bC4119a5',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:13]: https://basescan.org/address/0x4ccb0bb02fcaba27e82a56646e81d8c5bc4119a5

File: apps/web/src/addresses/usernames.ts [hunk 2]
```diff
@@ -54,13 +54,13 @@ export const BUILDATHON_ERC721_DISCOUNT_VALIDATOR: AddressMap = {
 };
 
 export const USERNAME_BASE_ETH_HOLDERS_DISCOUNT_VALIDATORS: AddressMap = {
-  [baseSepolia.id]: '0x',
-  [base.id]: '0x',
+  [baseSepolia.id]: '0xA475f24BEa985Ff66c1F0d9D8C23661215418894',
+  [base.id]: '0x55564490a44FDC2aEEa54B60eB1c79F124FD88b9',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:58]: https://basescan.org/address/0x55564490a44FDC2aEEa54B60eB1c79F124FD88b9

File: apps/web/src/addresses/usernames.ts [hunk 3]
```diff
@@ -54,13 +54,13 @@ export const BUILDATHON_ERC721_DISCOUNT_VALIDATOR: AddressMap = {
...
 };
 
 export const USERNAME_1155_DISCOUNT_VALIDATORS: AddressMap = {
   [baseSepolia.id]: '0xE41Cd25f429E10744938d5048646E721ac630aF3',
-  [base.id]: '0x',
+  [base.id]: '0x55246A2AE466257B2fB54d4BB881Fb3f17D8e03e',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:63]: https://basescan.org/address/0x55246A2AE466257B2fB54d4BB881Fb3f17D8e03e

File: apps/web/src/addresses/usernames.ts [hunk 4]
```diff
@@ -84,5 +84,5 @@ export const UNISWAP_USDC_WETH_POOL: AddressMap = {
 
 export const EXPONENTIAL_PREMIUM_PRICE_ORACLE: AddressMap = {
   [baseSepolia.id]: '0x2B73408052825e17e0Fe464f92De85e8c7723231',
-  [base.id]: '0x',
+  [base.id]: '0xd53B558e1F07289acedf028d226974AbBa258312',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:87]: https://basescan.org/address/0xd53B558e1F07289acedf028d226974AbBa258312

File: apps/web/src/addresses/usernames.ts [hunk 5]
```diff
@@ -54,13 +54,13 @@ export const BUILDATHON_ERC721_DISCOUNT_VALIDATOR: AddressMap = {
 };
 
 export const USERNAME_BASE_ETH_HOLDERS_DISCOUNT_VALIDATORS: AddressMap = {
-  [baseSepolia.id]: '0x',
-  [base.id]: '0x',
+  [baseSepolia.id]: '0xA475f24BEa985Ff66c1F0d9D8C23661215418894',
```
Comment (dneilroth) [inline apps/web/src/addresses/usernames.ts:57]: https://sepolia.basescan.org/address/0xa475f24bea985ff66c1f0d9d8c23661215418894

**Review by stevieraykatz — APPROVED** [review]

**Review by JFrankfurt — APPROVED** [review]

**Pull request merged**